src/cosl/coordinated_workers/coordinator.py

```diff
@@ -250,7 +251,7 @@ def __init__(
 
         self._is_coherent = is_coherent
         self._is_recommended = is_recommended
-        self._tracing_receivers_getter = tracing_receivers
+        self._tracing_receivers_getter = tracing_receivers or self._tracing_receivers_urls
```

Comment:
not sure we need it to be a getter: is there any chance it'll ever be dynamic? I feel every coordinator can hardcode it, it doesn't need to be a callback